phpBB/config/default/container/services.yml

```diff
@@ -8,13 +8,16 @@ imports:
     - { resource: services_db.yml }
     - { resource: services_event.yml }
     - { resource: services_feed.yml }
+    - { resource: services_files.yml }
```

Comment:
I think that if you merge the two, that would make sense, so I think we should only change the file names if the services don't go together logically.